### PR TITLE
Change the renaming of joint validate HT fields for VCF

### PR DESCRIPTION
**Pull request by jkgoodrich**
Test: eff2565dc9634e70a7166948cf1ca5c7

It would need a fix for ordering, but just a proposal

### CODE REVIEW

**Comment by KoalaQin** [comment]
I like your renaming idea, which will the header and export easier. I have some suggestions in PR #607

**Review by mike-w-wilson — CHANGES_REQUESTED** [review]
One thing with adding the exome and genome globals to the validated HT output but once that is in, LGTM! @KoalaQin , I presume you'll be taking this work over from Julia?

**Comment by KoalaQin** [comment]
Yep, happy to take over! Give me a moment.

File: gnomad_qc/v4/create_release/validate_and_export_vcf.py [hunk 1]
```diff
@@ -1170,12 +1208,19 @@ def main(args):
...
+                    info_expr = validate_hts[dt][ht.key].info
+                    info_expr = info_expr.select(
+                        *[f for f in info_expr if f not in in_joint_ht]
+                    )
+                    ht = ht.annotate(info=ht.info.annotate(**info_expr))
+                    ht = ht.annotate_globals(
+                        **{
+                            f"{dt}_{f}": validate_hts[dt].globals[f]
+                            for f in validate_hts[dt].globals
+                        }
```
Comment (mike-w-wilson) [inline gnomad_qc/v4/create_release/validate_and_export_vcf.py:1222]: This is already inside a datatype for loop and globals have the prefix
```suggestion
                    ht = ht.annotate_globals(**validate_hts[dt].index_globals())
```

**Review by mike-w-wilson — CHANGES_REQUESTED** [review]

File: gnomad_qc/v4/create_release/validate_and_export_vcf.py (hunk 1 above)
Comment (KoalaQin) [reply gnomad_qc/v4/create_release/validate_and_export_vcf.py]: It was under testing, now should work. Keep forgetting `index_globals()`.

**Comment by mike-w-wilson** [comment]
I've tested this on all datatypes and the output validated HT looks good!

**Review by mike-w-wilson — APPROVED** [review]
LGTM

**Comment by mike-w-wilson** [comment]
Let's not run this until all code updates are in for the joint VCF export in case we find something we need to update when writing the header and export code.

**Pull request merged**